src/cephadm/cephadm

```diff
@@ -2749,6 +2743,18 @@ def get_ceph_volume_container(ctx: CephadmContext,
     )
 
 
+def set_pids_limit_unlimited(ctx: CephadmContext, container_args: List[str]) -> List[str]:
+    # set containers pid-limits to unlimited rather than defaults (Docker 4096 / Podman 2048)
+    # Useful for daemons like iscsi where the default pid-limit limits the number of luns
+    # per iscsi target or rgw where increasing the rgw thread pool size to a value near
+    # the default pid-limit may cause the container may crash.
```

Comment:
The actual config option name is useful because one can grep for it:
```suggestion
    # per iscsi target or rgw where increasing rgw_thread_pool_size to a value near
    # the default pids-limit may cause the container to crash.
```